app/src/main/res/navigation/nav_graph.xml

```diff
@@ -316,4 +313,38 @@
         app:popUpTo="@id/nav_graph"
         app:popUpToInclusive="true" />
 
+    <navigation
+        android:id="@+id/nav_seeker_create_request"
+        android:label="create"
+        app:startDestination="@id/seekerCreateRequestEnterArticlesFragment">
+        <fragment
+            android:id="@+id/seekerCreateRequestEnterArticlesFragment"
+            android:name="app.nexd.android.ui.seeker.create.articles.SeekerCreateRequestEnterArticlesFragment"
+            android:label="@string/seeker_request_create_heading"
+            tools:layout="@layout/fragment_seeker_create_request_enter_articles">
+            <action
+                android:id="@+id/action_seekerCreateRequestEnterArticlesFragment_to_seekerCreateRequestConfirmAddressFragment"
+                app:destination="@id/seekerCreateRequestConfirmAddressFragment"
+                app:enterAnim="@animator/slide_in_right_to_left"
+                app:exitAnim="@animator/slide_out_right_to_left"
+                app:popEnterAnim="@animator/slide_in_left_to_right"
+                app:popExitAnim="@animator/slide_out_left_to_right" />
+        </fragment>
+        <fragment
+            android:id="@+id/seekerCreateRequestConfirmAddressFragment"
+            android:name="app.nexd.android.ui.seeker.create.address.SeekerCreateRequestConfirmAddressFragment"
+            android:label="SeekerCreateRequestConfirmAddressFragment"
+            tools:layout="@layout/fragment_seeker_create_request_confirm_address">
+            <action
+                android:id="@+id/action_seekerCreateRequestConfirmAddressFragment_to_seekerOverviewFragment"
+                app:destination="@id/seekerOverviewFragment"
+                app:enterAnim="@animator/slide_in_right_to_left"
+                app:exitAnim="@animator/slide_out_right_to_left"
+                app:popEnterAnim="@animator/slide_in_left_to_right"
+                app:popExitAnim="@animator/slide_out_left_to_right"
+                app:popUpTo="@id/seekerOverviewFragment"
```

Comment:
after confirming the delivery data, user should land in the seeker overview fragment

Reply:
the user will directed to the SeekerOverViewFragment. The popUpTo is included so pressing back doesn't bring the user back to the address confirmation screen.